acados/utils/math.c

```diff
@@ -1093,7 +1093,7 @@ reconstruct_A(A, V, d);
 }*/
 
 /* mirroring regularization */
-void regularize(int_t dim, real_t *A)
+void mirror(int_t dim, real_t *A, double epsilon)
 {
     real_t *V = (real_t *) calloc(dim * dim, sizeof(real_t));
     real_t *d = (real_t *) calloc(dim, sizeof(real_t));
```

Comment:
same here, calloc should be removed

Reply:
uuuups, totally missed that this happens online :)